src/redux/session.js

```diff
@@ -14,9 +14,11 @@ const Types = keyMirror({
 
 const banGoodListPaths = [
     '/accounts/banned-response',
+    '/accounts/bad-username',
     '/community_guidelines',
     '/privacy_policy',
-    '/terms_of_use'
+    '/terms_of_use',
+    '/accounts/update_username'
```

Comment:
This shouldn't be necessary here-- because this is a background request and not a user visited page. It should be added to the scratchr2 whitelist (that may or may not have been done already, can't see because it is private). But I think it may have been a mistake to add it here